plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -266,6 +266,12 @@ public void dropSchema(ConnectorSession session, String schemaName)
         execute(session, "DROP DATABASE " + quoted(schemaName));
     }
 
+    @Override
+    public void renameSchema(ConnectorSession session, String schemaName, String newSchemaName)
+    {
+        throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming schemas");
```

Comment:
I'll add a comment (and create follow up PR).

It's only supported by the "Atomic" database type (which seems to be default in newer versions - since `v20.10.2.20-stable`) but our testing version seems to use "Ordinary" database type and I was unable to find a way to check the database type in `renameSchema`.

Reply:
https://github.com/trinodb/trino/issues/10558